build.gradle

```diff
@@ -69,6 +69,18 @@ task installRealmJava(type:Task) {
     description = 'Install the Realm library and Gradle plugin into mavenLocal()'
 }
 
+task installRobolectric(type:GradleBuild) {
+    description = 'Install the Robolectric library into example'
```

Comment:
@zaki50 What version of realm-core do you use? I guess you are not using v0.95.7, or you installed another version on your `/usr/local/include`.
https://github.com/realm/realm-core/blob/fec030131b822f388b37630d60d1573aeb6a30d8/src/realm/table.hpp#L1420